org.freedesktop.Sdk.Extension.dotnet6.yaml

```diff
@@ -25,63 +25,67 @@ modules:
         sha256: 90337653d92d4a13de590781371c604f9031cdb50520366aa1e3a91e1efb1017
       - type: script
         commands:
-          - 'autoreconf -sif'
+          - autoreconf -sif
         dest-filename: autogen.sh
 
   - name: dotnet-sdk
     buildsystem: simple
     build-commands:
-      - 'mkdir -p /usr/lib/sdk/dotnet6/{bin,lib}'
-      - 'cp -r * /usr/lib/sdk/dotnet6/lib'
-      - 'ln -s /usr/lib/sdk/dotnet6/lib/dotnet /usr/lib/sdk/dotnet6/bin/dotnet'
+      - mkdir -p /usr/lib/sdk/dotnet6/{bin,lib}
+      - cp -r * /usr/lib/sdk/dotnet6/lib
+      - ln -s /usr/lib/sdk/dotnet6/lib/dotnet /usr/lib/sdk/dotnet6/bin/dotnet
     sources:
       - type: archive
         only-arches: [x86_64]
-        url: https://download.visualstudio.microsoft.com/download/pr/17b6759f-1af0-41bc-ab12-209ba0377779/e8d02195dbf1434b940e0f05ae086453/dotnet-sdk-6.0.100-linux-x64.tar.gz
+        url: https://dotnetcli.blob.core.windows.net/dotnet/Sdk/6.0.100/dotnet-sdk-6.0.100-linux-x64.tar.gz
         sha512: cb0d174a79d6294c302261b645dba6a479da8f7cf6c1fe15ae6998bc09c5e0baec810822f9e0104e84b0efd51fdc0333306cb2a0a6fcdbaf515a8ad8cf1af25b
         x-checker-data:
           type: html
           url: https://dotnetcli.blob.core.windows.net/dotnet/Sdk/6.0/latest.version
-          version-pattern: "^([\\d\\.a-z-]+)$"
-          url-template: "https://dotnetcli.blob.core.windows.net/dotnet/Sdk/$version/dotnet-sdk-$version-linux-x64.tar.gz"
+          version-pattern: ^([\d\.a-z-]+)$
+          url-template: https://dotnetcli.blob.core.windows.net/dotnet/Sdk/$version/dotnet-sdk-$version-linux-x64.tar.gz
+        sha256: 8489a798fcd904a32411d64636e2747edf108192e0b65c6c3ccfb0d302da5ecb
```

Comment:
flatpak-external-data-checker doesn't support checksums other than sha256 [yet](https://github.com/flathub/flatpak-external-data-checker/pull/208), so it'll ignore any other checksum set and add `sha256`.
For now, if you want to use f-e-d-c, you'll need to replace `sha512` with `sha256`.

Reply:
this checksum is not used by x-checker-data, right?

Reply:
`sha512` is ignored, `sha256` is used.